### DISCUSSION TITLE
blog/upgrade-blog-to-nextjs-141

### DISCUSSION THREAD

**Discussion by giscus** [shenlu89/shenlu.me#50] [Announcements]
# blog/upgrade-blog-to-nextjs-141

Upgrade Blog to Next.js 14.1

https://www.shenlu.me/blog/upgrade-blog-to-nextjs-141

**Comment by zawa8** [comment]
1. нeललo सheन,
टeसटińg iफ giसcuस coममeनट wiलल àपपeàर нeरe.
ińgलiसh31 : a-z + 5àԃнт
iनडiàन_लàनguageस_11 : लocaल फoनट gलयपhस फoर  5टडपबस+यरलमनफ  #vowelहiनदi

J. нello ㅅheㄴ,
ㅌeㅅㅌińg if giㅅcuㅅ coㅁㅁeㄴㅌ will àㅍㅍeà라 нe라e.
ińgliㅅh31 : a-z + 5àԃнт
iㄴ다iàㄴ_làㄴguageㅅ_11 : local foㄴㅌ glyㅍhㅅ fo라  5ㅌ다ㅍ바ㅅ+y라lㅁㄴf 
#korean zava8.github.io -> font -> koreàn115.ttf

**Comment by zawa8** [comment]
нello shen,

please **нelp** , i cloned your site ànd tryińg to comment when it is not gettińg displayed like it is dsplayed нere. please нelp for properly settińg **giscus** or provide steps for this on [reddit page for upgrade_blog_to_nextjs](https://www.reddit.com/r/reactjs/comments/1b40882/upgrade_blog_to_nextjs_v1410_with_mdx_tailwindcss/)
[cloned blog](https://aaztk.vercel.app/blog/setting-up-ubuntu-development-environment-with-docker-on-macos)
[cloned gitнub repo : zava810/shenlu](https://github.com/zava810/shenlu)

**Reply by zawa8** [reply]
now issue got resolved after adding all 4 enviroment variable from local .env file to vercel project enviroment variables : 
// For Giscus
NEXT_PUBLIC_REPO="paste this value in vercel project enviroment variable"
NEXT_PUBLIC_REPO_ID="paste this value in vercel project enviroment variable"
NEXT_PUBLIC_CATEGORY="paste this value in vercel project enviroment variable"
NEXT_PUBLIC_CATEGORY_ID="paste this value in vercel project enviroment variable"
